simulation/src/main/kotlin/it/unibo/collektive/examples/spreading/DistanceToSource.kt

```diff
@@ -0,0 +1,25 @@
+package it.unibo.collektive.examples.spreading
+
+import it.unibo.collektive.aggregate.api.Aggregate
+import it.unibo.collektive.stdlib.spreading.hopGradientCast
+
+/** Defined a data class to represent the association between a source
+ * node and its distance. */
```

Comment:
`[...] association between a source node and its distance.`

Its distance to what? Please document it better.